friends/src/components/FriendAdd.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { BrowserRouter as Router, Route, Link, NavLink } from "react-router-dom";
+
+function FriendAdd(props) {
+    console.log(props)
+  return (
+    <div>
+        <h2>ADD NEW FRIEND HERE !!!</h2>
+      <form onSubmit={props.addFriend}>
```

Comment:
You have access to props.history here to make this one click to add friend and return to the home page. example for line 9 `<form onSubmit={e=> {props.addFriend(e); props.history.push('/');}}>`